src/Altinn.Notifications.Core/Services/Interfaces/IGetOrderService.cs

```diff
@@ -20,5 +20,5 @@ public interface IGetOrderService
     /// </summary>
     /// <param name="senderRef">The senders reference</param>
     /// <param name="creator">The creator of the orders</param>
-    public Task<(List<NotificationOrder> Orders, ServiceError? Error)> GetOrdersBySendersReference(string senderRef, string creator);
+    public Task<(List<NotificationOrder>? Orders, ServiceError? Error)> GetOrdersBySendersReference(string senderRef, string creator);
```

Comment:
When working with collections we should try to always have an instance, but allow for it to be empty.